Source/WebCore/platform/network/ResourceResponseBase.h

```diff
@@ -232,6 +233,7 @@ class ResourceResponseBase {
     };
 
     WEBCORE_EXPORT ResourceResponseBase();
+    WEBCORE_EXPORT ResourceResponseBase(URL&&, String&& mimeType, long long expectedLength, const String& textEncodingName);
```

Comment:
Should we use String&& textEncodingName?

Reply:
Yes!